src/sentry/testutils/asserts.py

```diff
@@ -14,11 +16,10 @@ def assert_mock_called_once_with_partial(mock, *args, **kwargs):
         assert m_kwargs[kwarg] == kwargs[kwarg], (m_kwargs[kwarg], kwargs[kwarg])
 
 
-def assert_commit_shape(commit):
-    from sentry.models import CommitFileChange
+commit_file_type_choices = {c[0] for c in CommitFileChange._meta.get_field("type").choices}
```

Comment:
These imports are no longer causing cycles, so I'm putting this code back the way used to be.